kod/object/item/passitem/spelitem.kod

```diff
@@ -464,5 +465,10 @@ messages:
       return Send(self, @IsLabelled);
    }
 
+   GetTargetsBySpell(oSpell = $,who = $,lTargets = $)
```

Comment:
Add a comment explaining what this does and why subclasses might want to override it.